Frameworks/EOF/ERRest/Sources/er/rest/format/ERXXmlRestWriter.java

```diff
@@ -15,6 +15,8 @@
 /**
  *
  * @property ERXRest.suppressTypeAttributesForSimpleTypes (default "false") If set to true, primitive types, like type = "datetime", won't be added to the output
+ * @property <code>er.rest.ERXRest.includeNullValues</code> Boolean property to enable null values in return. Defaults
```

Comment:
Property is named _ERXRest.includeNullValues_ not _er.rest.ERXRest.includeNullValues_.